### ISSUE TITLE
[RFC] New, Renewable TCetiE Seaweed route

### ISSUE THREAD

**Issue by EmeraldsEmerald** [GTNewHorizons/GT-New-Horizons-Modpack#4823]
Hello there! This is a long issue, so please strap in.

Currently, the material requirements for T Ceti E Seaweed are incredibly high just for bio-circuits, requiring hundreds of thousands of the Seaweed just to produce a few circuits. (refer to [#3852](https://github.com/GTNewHorizons/NewHorizons/issues/3852)) This is only exacerbated by the lack of ANY good method to mass-produce the amounts of Seaweed, considering the fact the best choice is a ton of OpenComputer robots whacking away with shears.

This is an alternative option brainstormed by several people, including Me, ScriptedPiky, MamiTomoe, and Redfire. This method would rely on Bartworks' TCetiEis Fucus Serratus bacteria to produce the seaweed industrially.

The first step would be producing Agar. Agar would be a HV-tier item, produced like so:

2 Mince Meat + 2 Bones + 500L Dilute Sulfuric Acid => 1 Collagen + 500L Water
Chemical Reactor, HV-tier, 40 seconds per.
2 Collagen + 1000L Water + 1000L Phosphoric Acid => 1000L Gelatin Mixture
Chemical Reactor, HV-tier, 20 seconds per.
2000L Gelatin Mixture => 1 Phosphorus Dust, 1 Gelatin
Chemical Dehydrator, HV-tier, 30 seconds per.
1 Gelatin + 1000L Distilled Water => 1 Agar
Mixer, HV-tier, 30 seconds per.

This would come out at exactly 3 minutes for 1 Agar. The Phosphorus dust output only gives back half of what you put in via Phosphoric acid, assuming the P4O10 method is chosen for production.

The next part is renewable Unknown Liquid/Unknowwater production. You can pump it out of Barnarda C, but a renewable source would also prove handy.

1 Mysterious Crystal Dust + 1 Tiny Pile of Mytryl Dust + 1000L Saltwater => 1000L Unknowwater
1 Mysterious Crystal Dust + 8 Space Combs + 1000L Saltwater => 1000L Unknowwater
1 Mysterious Crystal Dust + 16 Space Flowers + 1000L Saltwater => 1000L Unknowwater
All three: LCR, UV-tier, 16 seconds per.

This gives a use to mytryl, and allows renewing Unknowwater. If mytryl isn't to a player's taste, they can also use flowers or combs. This is a nice alternative to pumping out a whole lot of it out of Barnarda C, especially if there isn't any Unknowwater left near you. This can allow for multiple playstyles, which leads to flexibility and enjoyment.

Next is a quick recipe - meat extract. This provides basically everything for bacteria to grow.

Any Raw Meat => 36L Meat Extract
Fluid Extractor, MV-tier, 15 seconds per.

Meat Extract would be 144L per cell, to better fit with Mince Meat being an actual dust.

The next step is the real "meat" of the suggestion - the production of Unknown Nutrient Agar. This would be a fluid - with its use being the actual creation of the Seaweed. Well, kind of.

3000L Phthalic Acid + 288L Meat Extract + 4000L Unknowwater + 8 Agar + 16 Stemcells + 64 Salt => 1000L Unknown Nutrient Agar
LCR, UV-tier, 80 seconds per.

There it is. Each value was selected for a given reason. The Unknown Nutrient Agar would be used as a way to breed the Bacteria, which would be done in a Bacterial Vat!

This Bacterial Vat recipe would create Seaweed Broth - this is what all the seaweed would be extracted from. 

8 Energy Crystal Dust + 1 Mytryl dust + 64 Seaweed (Pam's HC) +  1000L. Unknown Nutrient Agar + 1 Petri Dish (TCetiEis Fucus Serratus) => 100L Seaweed Broth
Bacterial vat, UV-tier. 60 seconds, Glass Tier - 8, Minimum Sieverts: 100

The seaweed broth is what seaweed would be extracted from. The vat CAN go up to 100 cells per extraction if you maintain exactly half of the output hatch full, sure, but that first needs a LOT of buildup of fluid and then maintaining that level. It should be perfectly fine in my eyes.

Finally, the Seaweed Broth would be Centrifuged!

1000L Seaweed Broth + Configured Circuit (Configs 1 through 6) => 64 T Ceti E seaweed
Centrifuge, UV-tier, 8 seconds.

The reason the Configured Circuit is included in here is to allow any type of seaweed to be produced. Config 1 would make the first type of Seaweed, config 2 another one, etc.

In case you're curious - a max output would not even produce enough for one tenth of a UEV circuit. The output seems fine.

~~Of course, this method would be best if replication of TCetiEis Fucus Serratus were possible. As it stands, it's not in the game, so bart would have to add it to BartWorks. Once that's done, the method is complete!~~

~~*EDIT For Clarification: Yes, the bacteria TCetiEis Fucus Serratus IS in the game! However, replicating it via liquid DNA or other methods used by other bacteria doesn't appear to possible. This is what would need to be added, not the bacteria itself.~~

Disregard above paragraphs, the bacteria isn't consumed

This is, of course, an RFC issue, so... please comment your opinions on this process.

**Comment by draknyte1** [comment]
Several folks including myself weighed in on this and put a lot of balance into the idea.

The only issue I personally see is the 1000x bonus on the Vat output when balanced.. But that's moreso a weird output issue with the Vat itself and not this recipe. Having the option to specify an bonus output % for recipes would alleviate this. If it's already a thing, neat.

**Comment by EmeraldsEmerald** [comment]
>The only issue I personally see is the 1000x bonus on the Vat output when balanced.. But that's moreso a weird output issue with the Vat itself and not this recipe. Having the option to specify an bonus output % for recipes would alleviate this. If it's already a thing, neat.

While that can seem like an issue, do remember: you need to fill at least a ZPM output hatch exactly halfway with the 100L recipe before the 1000x bonus kicks in. That is a ludicrous amount of processing, and that means the startup cost for this process is way bigger. In my eyes, it's fine.

**Comment by bartimaeusnek** [comment]
its actally not a "bonus", but the intended amount of stuff, where the recipes are balanced against. The completely filled/completely empty hatch is a punishment for bad automation and not the amount of stuff the vat should be balanced against.

**Comment by Anarack** [comment]
For anyone wonder how this affects the costs of bio circuits by napkin math on the bio material side
Best case with perfect vat matching raises eu cost of the bio components by 4%
Worse case with a empty vat hatch raises eu cost of the bio components by 147%

So automate well, not much of a problem, fail at automation, bio circuits will not be fun but still are craftable.

**Comment by observeroftime02** [comment]
I like it a lot. For those of us late in the game, there needs to desperately exist any method to get this stuff, because harvesting with shears when one is UV+ is pure cancer. Not an issue to gather a bit of it that way, but in order to get industrial amounts it's a pain in the neck. So this method introduces a way to not have to run around underwater with shears, I'm for it.

**Comment by bartimaeusnek** [comment]
@observeroftime02 the general idea behind that was using an OC robot with shears....

**Comment by observeroftime02** [comment]
@bartimaeusnek if that was the idea, it would be nice to have some sort of guide or tutorial for it, because OC is a mod that is extremely daunting to a lot of us, and getting into it without any sort of instruction is near impossible. If you have some handy guides or tutorials at hand I'd like to see them, as I'm interested myself. What I found on the internet about it was pretty sparse, and mostly revolves around pretty useless gimmicks such as making clocks etc.

**Comment by Spluff5** [comment]
This looks so cool and makes so much more sense than my, rather unrefined ideas.

**Comment by EmeraldsEmerald** [comment]
it's in! closing issue
